desktop-app/app/components/WebView/index.js

```diff
@@ -660,6 +663,7 @@ class WebView extends Component {
       screenshotInProgress,
     } = this.state;
     const deviceStyles = {
+      outline: (settings.get(USER_PREFERENCES) || {}).deviceOutlineStyle,
```

Comment:
@diego-vieira Can you pls get this value from the props instead of directly from the `electron-settings`? 

`this.props.browser.userPreferences` will have the preferences object.

Reply:
Thanks for pointing that out, I hope this looks good now.